### PR TITLE
Add Flame

### PR DESCRIPTION
**Pull request by thekillerregs**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/flame/background.png
Comment (maaaathis) [inline minecraft_servers/flame/background.png:1]: Compress this file

File: minecraft_servers/flame/background@2x.png
Comment (maaaathis) [inline minecraft_servers/flame/background@2x.png:1]: Compress this file

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/flame/background.png
Comment (thekillerregs) [reply minecraft_servers/flame/background.png:1]: Done!

File: minecraft_servers/flame/background@2x.png
Comment (thekillerregs) [reply minecraft_servers/flame/background@2x.png:1]: Done!

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**